### ISSUE TITLE
Mobility documentation is corrupted

### ISSUE THREAD

**Issue by sedubois** [shioyama/mobility#508]
The README [links for example](https://github.com/shioyama/mobility#fallbacks) to [this doc page](https://www.rubydoc.info/gems/mobility/Mobility/Configuration) but it shows the errors below:

<details>

```
SQLite3::CorruptException: database disk image is malformed
/usr/local/bundle/ruby/2.7.0/gems/sqlite3-1.4.2/lib/sqlite3/resultset.rb:108:in `step'
/usr/local/bundle/ruby/2.7.0/gems/sqlite3-1.4.2/lib/sqlite3/resultset.rb:108:in `next'
/usr/local/bundle/ruby/2.7.0/gems/sqlite3-1.4.2/lib/sqlite3/resultset.rb:133:in `each'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:334:in `block in fetch_rows'
/usr/local/bundle/ruby/2.7.0/gems/sqlite3-1.4.2/lib/sqlite3/database.rb:339:in `query'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:200:in `block (2 levels) in _execute'
/usr/local/bundle/ruby/2.7.0/gems/skylight-core-4.2.0/lib/skylight/core/probes/sequel.rb:22:in `block (2 levels) in log_connection_yield'
/usr/local/bundle/ruby/2.7.0/gems/activesupport-6.0.3.1/lib/active_support/notifications.rb:180:in `block in instrument'
/usr/local/bundle/ruby/2.7.0/gems/activesupport-6.0.3.1/lib/active_support/notifications/instrumenter.rb:24:in `instrument'
/usr/local/bundle/ruby/2.7.0/gems/activesupport-6.0.3.1/lib/active_support/notifications.rb:180:in `instrument'
/usr/local/bundle/ruby/2.7.0/gems/skylight-core-4.2.0/lib/skylight/core/probes/sequel.rb:16:in `block in log_connection_yield'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/database/logging.rb:38:in `log_connection_yield'
/usr/local/bundle/ruby/2.7.0/gems/skylight-core-4.2.0/lib/skylight/core/probes/sequel.rb:15:in `log_connection_yield'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:200:in `block in _execute'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/connection_pool/threaded.rb:92:in `hold'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/database/connecting.rb:270:in `synchronize'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:193:in `_execute'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:139:in `execute'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/dataset/actions.rb:1089:in `execute'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/adapters/sqlite.rb:327:in `fetch_rows'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/dataset/actions.rb:940:in `with_sql_each'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/dataset/actions.rb:950:in `with_sql_first'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/dataset/placeholder_literalizer.rb:192:in `first'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/dataset/actions.rb:243:in `first'
/usr/local/bundle/ruby/2.7.0/gems/sequel-5.28.0/lib/sequel/model/plugins.rb:33:in `first'
/app/lib/gem_store.rb:23:in `[]'
/usr/local/bundle/ruby/2.7.0/bundler/gems/yard-ca59056a5c70/lib/yard/server/router.rb:83:in `parse_library_from_path'
/usr/local/bundle/ruby/2.7.0/bundler/gems/yard-ca59056a5c70/lib/yard/server/router.rb:112:in `route'
/usr/local/bundle/ruby/2.7.0/bundler/gems/yard-ca59056a5c70/lib/yard/server/router.rb:56:in `call'
/usr/local/bundle/ruby/2.7.0/bundler/gems/yard-ca59056a5c70/lib/yard/server/rack_adapter.rb:52:in `call'
/app/app.rb:405:in `block (2 levels) in <class:DocServer>'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1636:in `call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1636:in `block in compile!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:987:in `block (3 levels) in route!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1006:in `route_eval'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:987:in `block (2 levels) in route!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1035:in `block in process_route'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1033:in `catch'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1033:in `process_route'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:985:in `block in route!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:984:in `each'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:984:in `route!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1098:in `block in dispatch!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `block in invoke'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `catch'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `invoke'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1095:in `dispatch!'
/usr/local/bundle/ruby/2.7.0/gems/skylight-core-4.2.0/lib/skylight/core/probes/sinatra.rb:36:in `dispatch!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:919:in `block in call!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `block in invoke'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `catch'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1072:in `invoke'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:919:in `call!'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:908:in `call'
/usr/local/bundle/ruby/2.7.0/gems/skylight-core-4.2.0/lib/skylight/core/middleware.rb:84:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/common_logger.rb:38:in `call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:231:in `call'
/app/app.rb:187:in `call'
/app/app.rb:30:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/head.rb:12:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/conditional_get.rb:27:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/xss_header.rb:18:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/path_traversal.rb:16:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/json_csrf.rb:26:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/base.rb:50:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/base.rb:50:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-protection-2.0.8.1/lib/rack/protection/frame_options.rb:31:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/null_logger.rb:11:in `call'
/usr/local/bundle/ruby/2.7.0/gems/rack-2.2.3/lib/rack/head.rb:12:in `call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:194:in `call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1951:in `call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1503:in `block in call'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1728:in `synchronize'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1728:in `synchronize'
/usr/local/bundle/ruby/2.7.0/gems/sinatra-2.0.8.1/lib/sinatra/base.rb:1503:in `call'
/usr/local/bundle/ruby/2.7.0/gems/puma-4.3.5/lib/puma/configuration.rb:228:in `call'
/usr/local/bundle/ruby/2.7.0/gems/puma-4.3.5/lib/puma/server.rb:713:in `handle_request'
/usr/local/bundle/ruby/2.7.0/gems/puma-4.3.5/lib/puma/server.rb:472:in `process_client'
/usr/local/bundle/ruby/2.7.0/gems/puma-4.3.5/lib/puma/server.rb:328:in `block in run'
/usr/local/bundle/ruby/2.7.0/gems/puma-4.3.5/lib/puma/thread_pool.rb:134:in `block in spawn_thread'
```

</details>

**Comment by shioyama** [comment]
Wow what's up with rubydocs.info? ... ok I guess I'll need to stop linking to them. Thanks for the heads up!

**Comment by shioyama** [comment]
Oh ok now it seems to be working, and I get this error:

```
Sorry, but we couldn't find the page /gems/mobility/Mobility/Configuration in the mobility gem. Are you sure that class, method, or file exists?
```

Which makes sense since the `Mobility::Configuration` class is now gone. I'll update the readme.

**Comment by shioyama** [comment]
Fixed in 687726d4bab774067da29efc7ade311a71260dcb
